### PR TITLE
update install doc for release/1.7.0

### PR DESCRIPTION
**Pull request by JepsonWong**
update install doc for release 1.7.0.

image

image

### CODE REVIEW

**Comment by adaxiadaxi** [comment]
LGTM

File: doc/fluid/install/index_cn.rst [hunk 1]
```diff
@@ -44,21 +44,21 @@
...
         * 如果您是使用本机 **pip** 安装：
 
-            * CentOS 7 支持 CUDA 9.0/9.2/10.0，不支持10.1，支持 CUDA 9.1 但仅支持单卡模式
-            * CentOS 6 支持 CUDA 9.0/9.1/9.2/10.0 单卡模式，不支持10.1
+            * CentOS 7 支持 CUDA 9.0/9.1/9.2/10.0/10.1，CUDA 9.1 仅支持单卡模式
+            * CentOS 6 支持 CUDA 9.0/9.1/9.2/10.0/10.1 单卡模式
         * 如果您是使用本机源码编译安装：
 
-            * CentOS 7 支持 CUDA 9.0/9.2/10.0
+            * CentOS 7 支持 CUDA 9.0/9.2/10.0/10.1
```
Comment (phlrain) [inline doc/fluid/install/index_cn.rst]: 9.1 是不是支持

Comment (JepsonWong) [reply doc/fluid/install/index_cn.rst]: done

**Review by phlrain — APPROVED** [review]

**Review by swtkiwi — APPROVED** [review]
LGTM

**Pull request merged**